index.js

```diff
@@ -2,14 +2,59 @@
 const execa = require('execa');
 const findVersions = require('find-versions');
 
-module.exports = (binary, options = {}) => {
-	return execa(binary, options.args || ['--version'])
-		.then(result => findVersions(result.stdout || result.stderr, {loose: true})[0])
-		.catch(error => {
+const knownBinaryArgs = new Map([
+	['openssl', ['version']],
+	['ffmpeg', ['-version']]
```

Comment:
Can you include some more popular binaries with non-`--version` version flags?

Reply:
To be honest, I can't remember any, not a type of information I try hard to remember, and everything I try of the top of my head accepts `--version`. But should probably put up a notice in readme incentivising people to file an issue if they find some.
I've added `ffprobe` and `ffplay` though.

Reply:
(for the notice part, it would really be best if you formulate it and put where you want, otherwise we will end up with a couple of more rounds of back and forward about one sentence)